apps/src/lib/node/ledger/mod.rs

```diff
@@ -353,16 +358,10 @@ async fn run_aux(config: config::Ledger, wasm_dir: PathBuf) {
                     abort_sender,
                 )
             }
-            #[cfg(feature = "eth-fullnode")]
-            ethereum_node::oracle::run_oracle(
-                ethereum_url,
-                eth_sender,
-                abort_sender,
-            )
```

Comment:
can this all be collpased into a single match case (more readable IMO), by changing the config struct to take an enum with the different options?